sdk/CorrelationContext.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk;
+
+use OpenTelemetry\Context\Context;
+use OpenTelemetry\Context\ContextKey;
+
+class CorrelationContext
+{
+    use Context;
+
+    /**
+     * @param CorrelationContext $context
+     */
+    public function getCorrelations($context = null)
+    {
+        // TODO: Write me
+    }
+
+    /**
+     * @param ContextKey $key
+     *
+     * @return CorrelationContext
+     */
+    public function removeCorrelation(ContextKey $key): CorrelationContext
+    {
+        if ($this->key === $key) {
+            return $this->parent;
+        }
+
+        $this->removeCorrelationHelper($key, null);
+
+        return $this;
+    }
+
+    /**
+     * @param ContextKey $key
+     * @param CorrelationContext|null $child
+     * @suppress PhanUndeclaredMethod
```

Comment:
I added this to appease the CI for now, but I think we might need to address it.